tls/tests/test_remote.py

```diff
@@ -15,6 +17,16 @@
 from datadog_checks.tls.tls import TLSCheck
 from datadog_checks.tls.tls_remote import TLSRemoteCheck
 
+try:
+    from ssl import SSLCertVerificationError
```

Comment:
If I were you I'd just import `from ssl import SSLCertVerificationError` locally in your test